app/graphql/types/anime.rb

```diff
@@ -1,4 +1,8 @@
 class Types::Anime < Types::BaseObject
   implements Types::Media
   implements Types::EpisodicInterface
+
+  field :youtube_video_id, String,
```

Comment:
I've been trying to generously rename fields to saner things in the GraphQL API.

What would you say to `youtube_trailer_video_id`?

(eventually I'd love to replace this with an Embed object for any video, since we have a concept of Embeds now, but that seems like a bigger, dreamier effort)

Reply:
Sounds good to me 👍